src/amo/constants.js

```diff
@@ -1,31 +1,4 @@
 /* @flow */
-// Action types.
```

Comment:
Sorry for the noise. It was bothering me that these action types were not in the action module. This cleans up `amo/constants` quite a bit.